node/tests/webView2Test.ts

```diff
@@ -0,0 +1,41 @@
+import { test as base } from '@playwright/test';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import childProcess from 'child_process';
+
+export const test = base.extend({
+  browser: async ({ playwright }, use, testInfo) => {
+    const cdpPort = 10000 + testInfo.workerIndex;
+    const executable = path.join(__dirname, '../../webview2-app/bin/Debug/net6.0-windows/webview2.exe');
```

Comment:
just curious why is it app/bin/Debug and not Release?

Reply:
default is Debug and Release would require `-c release` I guess users know what to do, will adjust it then accordingly to their setup.